File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');

let nextNum = 0;

if (!isNaN(input[0])) {
    nextNum = Number(input[0]) + 3;
} else if (!isNaN(input[1])) {
    nextNum = Number(input[1]) + 2;
} else if (!isNaN(input[2])) {
    nextNum = Number(input[2]) + 1;
}

if (nextNum % 3 === 0 && nextNum % 5 === 0) {
    console.log('FizzBuzz');
} else if (nextNum % 3 === 0) {
    console.log('Fizz');
} else if (nextNum % 5 === 0) {
    console.log('Buzz');
} else {
    console.log(nextNum);
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.py
a = input()
b = input()
c = input()

next_val = 0

if a.isdigit():
    next_val = int(a) + 3
elif b.isdigit():
    next_val = int(b) + 2
elif c.isdigit():
    next_val = int(c) + 1

if next_val % 3 == 0 and next_val % 5 == 0:
    print('FizzBuzz');
elif next_val % 3 == 0 and next_val % 5 != 0:
    print('Fizz');
elif next_val % 3 != 0 and next_val % 5 == 0:
    print('Buzz');
else:
    print(next_val)